src/shared/clock-util.c

```diff
@@ -152,7 +155,12 @@ int clock_apply_epoch(void) {
         } else
                 epoch_usec = timespec_load(&st.st_mtim);
 
-        if (now(CLOCK_REALTIME) >= epoch_usec)
+        now_usec = now(CLOCK_REALTIME);
+        diff_usec = usec_sub_unsigned(now_usec, epoch_usec);
+        if(!diff_usec)
```

Comment:
as per coding style: don't do numeric comparison with C's downgrade-to-bool feature. i.e. do an explicit check == 0 check.

also space after the `if` and before the `(`.

But really, don't use usec_sub_unsigned() here unless you know that the first param is larger than the second. It's ugly to rely on usec_sub_unsigned()'s internal handling of this case. i.e. make this check explicit, by comparing now_usec with epoch_usec first.